src/random.cpp

```diff
@@ -48,6 +61,51 @@ namespace libtorrent
 #endif
 			return rng;
 		}
+
+		void random_bytes(span<char> buffer)
```

Comment:
the problem is that `std::random_device` is not specified to generate cryptographically PSRN